core/src/main/scala/org/elasticmq/actor/queue/operations/SendMessageOp.scala

```diff
@@ -37,7 +37,7 @@ trait SendMessageOp extends Logging {
   }
 
   private def addMessage(message: NewMessageData) = {
-    val internalMessage = InternalMessage.from(message, queueData)
+    val internalMessage = InternalMessage.from(message, queueData, nextSequenceNumber())
```

Comment:
wouldn't it make more sense to adapt the message attributes here, and add the sequence number, before converting to an `InternalMessage`? That way we would avoid passing things around a bit

Reply:
👍